docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -6,85 +6,81 @@ description: Nextcloud integration setup
+As in the previous step, the OAuth values are generated automatically, but this time on the Nextcloud end. These are needed to allow the connection From Nextcloud to OpenProject to function, so you will need to once again copy them. Copy these values from Nextcloud to OpenProject (without closing the Nextcloud tab). If you had not clicked on the *Done. Continue setup* button from in previous step, you can now do so to see the screen where you will be able to paste the Nextcloud OAuth values in OpenProject.
 
 ![Nextcloud_NC_OAuth_values](3_2_04-NC_Step_3.png)
 
 ![OpenProject_NC_OAuth_values](3_3_01-OP_OAuth_application_details.png)
 
-Once the values are copied you can click the button "Save and complete setup" in the OpenProject window and the "Yes, I have copied these values" in the Nextcloud window. Once this is done your instance configuration will be completed.
+Once the values are copied, you can click the button *Save and complete setup* in your OpenProject tab and the *Yes, I have copied these values* in Nextcloud. Once this is done, your instance configuration is completed.
 
 ![OpenProject_success](3_4_00 - OAuth - OP Success.png)
 
 ![Nextcloud_success](3_2_05-NC_Success.png)
 
 **5. Enable the file storages module and activate it in the desired project** - in OpenProject
 
-Now the integration setup is complete and ready to use, but as administrator you still need to activate the file storages module and specify in which projects you need the file storage integration that you just set up.
+Now that the integration setup is complete and ready to use, there is just one more step for you the administrator: you need to activate the *File storages* module and specify for which projects you would like to enable your new Nextcloud integration.
 
-To activate the module please go to the Project settings in the desired project and access the Modules menu entry in the side menu, there you can activate the module "File storages". Once this is done a new lateral menu entry called "File storages" at the bottom of the menu where you can select which file storage integration you want to activate in this project.
+To activate the module, go to the *Project settings* in the desired project and access the Modules menu entry in the side menu. There, activate the "File storages" module. Once this is done, a new menu entry called "File storages" appears at the bottom, where you can select your new Nextcloud integration.
```

Comment:
Hi @psatyal,

I have the feeling that removing the part of the menu makes the sentence unclear (the bottom of the content? bottom of the complete screen?) I would like to keep the menu or the side menu there to make it clear.

Reply:
Hi @marcalcobe, just fixed this in a separate commit.